webapp/src/utils/jira_issue_metadata.tsx

```diff
@@ -205,8 +206,22 @@ export function getCustomFieldFiltersForProjects(metadata: IssueMetadata | null,
     return sortByName(result);
 }
 
-export function getCustomFieldValuesForProjects(metadata: IssueMetadata | null, projectKeys: string[]): ReactSelectOption[] {
-    return getCustomFieldsForProjects(metadata, projectKeys).map((field) => ({
+const avoidedCustomTypesForEvents = [
+    'com.pyxis.greenhopper.jira:gh-sprint',
+    'com.pyxis.greenhopper.jira:gh-lexo-rank',
```

Comment:
Michael, what's the reason for hardcoding these types?

Reply:
The initial implementation of events had hardcoded the available event options here:
https://github.com/mattermost/mattermost-plugin-jira/blob/ab10e6091ef2baefdd9cf0c7f3e88fb78fd281ee/webapp/src/components/modals/channel_settings/edit_channel_settings.tsx#L36-L46

Any field that is listed here that is also defined as a `custom` type by Jira (among this collection is just Sprint and Rank), will need to be explicitly ignored when we are parsing the issue metadata so we do not have duplicate options for the events.

Also, since these values have existed since v2.1, we cannot change the hardcoded identifiers, so we need them to stay the same value as they were originally introduced.

Reply:
Would be nice to move these somewhere more appropriate. Maybe after a good refactoring/cleanup.

Reply:
@cpoile you're right. They should at least be explicitly define into enums somewhere. They would make the code much more readable.